packages/web-react/src/components/UNSTABLE_EmptyState/demo/EmptyStateDefault.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { ButtonLink } from '../../Button';
+import { Heading } from '../../Heading';
+import { Link } from '../../Link';
+import { Text } from '../../Text';
+import { UNSTABLE_ActionLayout } from '../../UNSTABLE_ActionLayout';
+import UNSTABLE_EmptyState from '../UNSTABLE_EmptyState';
+import UNSTABLE_EmptyStateSection from '../UNSTABLE_EmptyStateSection';
+
+const EmptyStateDefault = () => (
+  <UNSTABLE_EmptyState spacing="space-700">
+    <UNSTABLE_EmptyStateSection>
+      <div style={{ border: '1px solid #E9E9E9', borderRadius: '10px', padding: '12px' }}>Icon</div>
```

Comment:
Can we use the actual Icon component here, please?

Reply:
It will be refactored in https://jira.almacareer.tech/browse/DS-1310